packages/bitcore-wallet-client/test/paypro.test.js

```diff
@@ -1,14 +1,14 @@
 'use strict';
 
-var _ = require('lodash');
-var chai = chai || require('chai');
-var sinon = sinon || require('sinon');
-var should = chai.should();
-var {PayPro: payPro} = require('../ts_build/lib/paypro');
-var TestData = require('./testdata');
+const chai = require('chai');
+const sinon = require('sinon');
+const crypto = require('crypto');
+const should = chai.should();
+const { PayPro: payPro } = require('../ts_build/lib/paypro');
+const TestData = require('./testdata');
 
 function mockRequest(bodyBuf, headers) {
-  bodyBuf = _.isArray(bodyBuf) ? bodyBuf : [bodyBuf];
+  bodyBuf = Array.isArray(bodyBuf) ? bodyBuf : [bodyBuf];
```

Comment:
This is fine. The spirit of this line could be stated as "If bodyBuf isn't an array, make it one". So I would propose:
if (!Array.isArray(bodyBuf)) {
  bodyBuf = [bodyBuf];
}

Admittedly completely subjective.